ui/dashboards/src/views/ViewDashboard/DashboardApp.tsx

```diff
@@ -44,6 +45,7 @@ export const DashboardApp = (props: DashboardAppProps) => {
         </ErrorBoundary>
         <PanelDrawer />
         {panelGroupDialog && <PanelGroupDialog />}
+        {deletePanelGroupDialog && <DeletePanelGroupDialog />}
```

Comment:
Nit: at some point (doesn't have to be now), I think we're going to want to stop conditionally rendering these dialogs and use their `open` prop so we get proper animation transitions (i.e. conditionally rendering means the dialog just disappears rather than transitioning out).